test/functional/feature_evm_contracts.py

```diff
@@ -438,6 +438,21 @@ def non_payable_proxied_contract(self):
             receipt["gasUsed"] * receipt["effectiveGasPrice"],
         )
 
+    # should fail since it is larger than DVM TX size limit
+    def fail_send_large_tx(self):
+        assert_raises_web3_error(
+            -32001,
+            "reason: scriptpubkey",
+            self.node.w3.eth.send_transaction,
+            {
+                "to": "0x0000000000000000000000000000000000000000",
+                "from": self.evm_key_pair.address,
+                "data": "0x" + "f" * (32000 * 5),
```

Comment:
Why not use `ff` so it's 1 byte. And then go right above 64k? (DVM Envelope would take up a bit more, but should be minimal for EVM TX)

Reply:
Makes sense, updated in 2c85aa1